.github/workflows/sphinx.yml

```
@@ -0,0 +1,67 @@
+name: sphinx-build
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  BUILD_TYPE: Release
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

For security reasons use full name